test/specs/commonTests/handlesAccessibility.tsx

```diff
@@ -1,38 +1,52 @@
 import * as React from 'react'
 
 import { getTestingRenderedComponent } from 'test/utils'
-import { ButtonBehavior, DefaultBehavior } from 'src/lib/accessibility'
+import { DefaultBehavior } from 'src/lib/accessibility'
+import { Accessibility, AriaRole } from 'src/lib/accessibility/interfaces'
+import { FocusZone } from 'src/lib/accessibility/FocusZone'
 
-const getProp = (renderedComponent, propName, partSelector) => {
+const getProp = (renderedComponent, propName, partSelector, isWrappedInFocusZone) => {
   const target = partSelector
     ? renderedComponent.render().find(partSelector)
     : renderedComponent.render()
 
-  return target.first().prop(propName)
+  let node = target.first()
+  if (isWrappedInFocusZone) {
```

Comment:
I would add here check for the node, as using partSelector might return nothing if it doesn't exists in the component. If the intention is for the test to fall in that case, I would rather handle that and throw explicit exception explaining what was the cause for the test to fail.

Reply:
Thanks! So I investigated and actually, `render()` [should return Cheerio objects](https://airbnb.io/enzyme/docs/api/ReactWrapper/render.html) and [Cheerio](https://cheerio.js.org/) has a chaining API which always returns another Cheerio object, meaning it can never be undefined and you can call functions even on empty Cheerio wrappers.

So what happens if no element is matched is that the whole `getProp` function would just return `undefined` from the last `.prop(...)` call, which is exactly how it worked before my PR.

I guess it's ok to keep it as it is?

Reply:
Yes, should be ok then.